.github/PULL_REQUEST_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,26 @@
+# Description
+
+Please provide a brief description of the changes made in this pull request.
+
+## Related Issue
+
+If this pull request is related to any issue, please mention it here. Additionally, make sure that the issue is assigned to you before submitting this pull request.
+
+## Checklist
+
+- [ ] I have tested the changes locally.
+- [ ] I have followed the project's coding style guidelines.
+- [ ] I have updated the documentation, if necessary.
+- [ ] I have added tests, if applicable.
```

Comment:
Do you think we should add one more line for "I signed the commits"

Reply:
good call